explorer/Explorer.tsx

```diff
@@ -65,6 +66,7 @@ interface ExplorerProps extends SerializedGridProgram {
 interface ExplorerPatchObject extends GrapherQueryParams {
     pickerSort?: SortOrder
```

Comment:
I moved these things to explorerProgram after our convo last week. hideControls should go on explorerProgram as well. My branch isn't merged in yet, so no biggie, just wanted to mention it.

Reply:
Alright, I'll wait for your merge then until I merge this in.